plaid/api.go

```diff
@@ -0,0 +1,31 @@
+package plaid
+
+import "os"
+
+// APIResponse is the base struct for all responses from the Plaid API.
+type APIResponse struct {
+	RequestID string `json:"request_id"`
+}
+
+// SecretAPIRequest is used to make private requests with a secret key.
+type SecretAPIRequest struct {
```

Comment:
I don't think there is a reason to export this struct right now @evanlimanto

Reply:
⚡️